yt_dlp/extractor/porndead.py

```diff
@@ -0,0 +1,127 @@
+import re
+import urllib.parse
+
+from yt_dlp.utils._utils import int_or_none
+
+from .common import InfoExtractor
+from ..utils import ExtractorError
+
+
+class PornDeadIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?porndead\.org/video/(?P<id>[0-9a-f]+)'
+    _TESTS = [
+        {
+            'url': 'https://porndead.org/video/65fefcb523810',
+            'info_dict': {
+                'id': '65fefcb523810',
+                'ext': 'mp4',
+                'title': 'Hysterical Literature - Isabel Love',
+                'age_limit': 18,
+            },
+        },
+    ]
+
+    def _real_extract(self, url):
+        url = url.strip().lower()
+
+        # if www is missing, add it because the relative URLs seem to depend on it
+        parsed = urllib.parse.urlparse(url)
+        if parsed.netloc == 'porndead.org':
+            parsed = parsed._replace(netloc='www.porndead.org')
+            url = urllib.parse.urlunparse(parsed)
+
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        # find video title ideally
+        title = (
+            self._html_search_regex(
+                r'<div[^>]+class=["\']title_video["\'][^>]*>([^<]+)</div>',
+                webpage,
+                'title',
+                default=None,
+            )
+            or self._og_search_title(webpage, default=None)
+            or f'Video {video_id}'
+        )
```

Comment:
1. Why not use the `<title>` element?
2. So consider `_html_extract_title()` method
3. Consider `_generic_title()` method as a last resort.